content/for-developers/tutorials/outer-loop/01-access-cluster/access-cluster.md

```diff
@@ -1,5 +1,21 @@
 # Access Your Cluster
```

Comment:
@mustafaStakater this is required in inner loop as well so, moved to outer loop section?

Reply:
should we keep it in both section or move it to how-to section

Reply:
also this document logs the user in with saap frontend, we will remove it for inner loop docs